curations/npm/npmjs/-/azure-maps-control.yaml

```diff
@@ -21,3 +21,6 @@ revisions:
   2.1.4:
     licensed:
       declared: OTHER
+  2.1.5:
+    licensed:
+      declared: NONE
```

Comment:
Looking at the preceding entries, looks like it should have been "OTHER". But I don't see a way for me to change this...